.github/workflows/ci.yml

```diff
@@ -104,20 +104,20 @@ jobs:
           make release
 
       - name: Publish nightly release to aptly
-        if: github.ref == 'refs/heads/master'
+        #if: github.ref == 'refs/heads/master'
```

Comment:
I think this line shouldn't be commented out. The nightlies should be created for the default branch (`master`), not for every branch and every PR

```suggestion
        if: github.ref == 'refs/heads/master'
```

Reply:
Yes sure ;) 
But as the behaviour is different in CI as on my local machine I need to test this somehow ...

- [ ] only publish from default branch